### PR TITLE
Make movie.json downloadable in one click

### PR DESCRIPTION
**Pull request by bidoubiwa**
wouhou 🎉 Be clicking on movies.json it should download it

fixes: #954

### CODE REVIEW

**Comment by dichotommy** [comment]
It doesn't work in the deploy preview (the downloaded file is called `movies.json`, but is actually a 404 page in HTML).

File: learn/getting_started/quick_start.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ To be processed, all documents must share one common <clientGlossary word="field
 
 There are [several ways to let MeiliSearch know what the primary key](/learn/core_concepts/documents.md#primary-key) is. The easiest one is to have an <clientGlossary word="attribute" /> that contains the string `id` in a case-insensitive manner.
 
-Below is an example to showcase how to add documents to an index called `movies`. To follow along, first right-click this link and download the file: [movies.json](https://raw.githubusercontent.com/meilisearch/MeiliSearch/master/datasets/movies/movies.json). Then, move the downloaded file to your working directory.
+Below is an example to showcase how to add documents to an index called `movies`. To follow along, first click this link and download the file: <a id="downloadMovie" href="./movies.json" download="movies.json">movies.json</a>. Then, move the downloaded file to your working directory.
```
Comment (bidoubiwa) [inline learn/getting_started/quick_start.md:66]:
```suggestion
Below is an example to showcase how to add documents to an index called `movies`. To follow along, first click this link and download the file: <a id="downloadMovie" href="/movies.json" download="movies.json">movies.json</a>. Then, move the downloaded file to your working directory.
```

File: learn/getting_started/quick_start.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ To be processed, all documents must share one common <clientGlossary word="field
 
 There are [several ways to let MeiliSearch know what the primary key](/learn/core_concepts/documents.md#primary-key) is. The easiest one is to have an <clientGlossary word="attribute" /> that contains the string `id` in a case-insensitive manner.
 
-Below is an example to showcase how to add documents to an index called `movies`. To follow along, first right-click this link and download the file: [movies.json](https://raw.githubusercontent.com/meilisearch/MeiliSearch/master/datasets/movies/movies.json). Then, move the downloaded file to your working directory.
+Below is an example to showcase how to add documents to an index called `movies`. To follow along, first click this link and download the file: <a id="downloadMovie" href="/movies.json" download="movies.json">movies.json</a>. Then, move the downloaded file to your working directory.
```
Comment (dichotommy) [inline learn/getting_started/quick_start.md:66]:
```suggestion
Below is an example to showcase how to add documents to an index called `movies`. To follow along, first click this link to download the file: <a id="downloadMovie" href="/movies.json" download="movies.json">movies.json</a>. Then, move the downloaded file to your working directory.
```

**Comment by dichotommy** [comment]
bors merge

**Pull request merged**